frontend/src/components/Checkbox/CheckboxOthers.tsx

```diff
@@ -0,0 +1,56 @@
+import { ChangeEvent, cloneElement, isValidElement, useRef } from 'react'
+import { CheckboxProps, Flex, forwardRef, useMergeRefs } from '@chakra-ui/react'
+import { createContext } from '@chakra-ui/react-utils'
+
+import { Checkbox } from './Checkbox'
+
+const [OthersProvider, useOthersContext] = createContext<{
+  onInputChange: () => void
+}>({
+  name: 'OthersContext',
+  strict: false,
+})
+
+export const CheckboxOthers = forwardRef<CheckboxProps, 'input'>(
+  ({ children, ...props }, ref) => {
+    const inputRef = useRef<HTMLInputElement | null>(null)
+    const mergedRef = useMergeRefs(ref, inputRef)
+
+    const handleInputChange = () => {
+      if (!inputRef?.current?.checked) {
+        inputRef?.current?.click()
+      }
+    }
+
+    return (
+      <OthersProvider value={{ onInputChange: handleInputChange }}>
```

Comment:
we probably don't need `OthersProvider` for this, just pass in `onInputChange` directly to `OthersWrapper` :P

```ts
<OthersWrapper onInputChange={handleInputChange}>
  ...
</OthersWrapper>
```

and have OthersWrapper take in an onInputChange prop in addition to other props